### DISCUSSION TITLE
Leaving code-summary empty

### DISCUSSION THREAD

**Discussion by andrewpbray** [quarto-dev/quarto-cli#2289] [Q&A] [answered]
I have been setting `code-folding: true` for my html doc, but I find the "Code" text next to the toggle triangle to be somewhat distracting (this is for a doc where 95% of readers won't/shouldn't care about the code).  I spotted that there's a `code-summary` option where you can pass it an alternative label like "Click here for code".

When I tried to leave it as an empty string I get:

```yaml
---
title: "Untitled"
format:
  html:
    code-fold: true
    code-summary: ""
---
```

```
Error running filter /Applications/quarto/share/filters/quarto-post/quarto-post.lua:
...cations/quarto/share/filters/quarto-post/quarto-post.lua:1119: attempt to index a nil value (field 'integer index')
stack traceback:
	...cations/quarto/share/filters/quarto-post/quarto-post.lua:2282: in function <...cations/quarto/share/filters/quarto-post/quarto-post.lua:2264>
```

`code-summary: "&nbsp"` doesn't do it either - it gets rendered as:

<img width="117" alt="Screen Shot 2022-09-04 at 8 37 08 AM" src="https://user-images.githubusercontent.com/1627362/188321646-311d5a0b-cd84-4312-a3b1-644a86231515.png">

I don't have a good hold on how yaml field values are parsed, but this is surprising behavior since something like `title: ""` works ok.

**Comment by cscheid** [comment, marked as answer]
That empty string error is a bug (not that it should necessarily hide it, but at the very least the error should be nicer.)
